src/SDK/Client/Platform/broadcastStateTransition.ts

```diff
@@ -32,9 +32,13 @@ export default async function broadcastStateTransition(platform: Platform, state
         if (e.data) {
             data = e.data;
         } else if (e.metadata) {
-            const errors = e.metadata.get('errors');
-            data = {};
-            data.errors = errors && errors.length > 0 ? JSON.parse(errors) : errors;
+            if (typeof e.metadata.get === 'function') {
```

Comment:
Could you please add comments to describe all these cases (e.data, e.details, e.metadata) and so on. It will be super helpful for my error framework research.

Reply:
@antouhou Are you aware about `e.data` and `e.details`?

Reply:
@shumkov I actually don't remember the exact cases in which those were happening, but it's there for a reason

Reply:
Ok, nevermind then.